tests/testthat/test-dataset.R

```diff
@@ -1,27 +1,50 @@
-context("Dataset object and methods") 
+context("Dataset object and methods")
 
 with(fake.HTTP, {
     test.ds <- loadDataset("test ds")
-    
+    today <- Sys.Date()
+    test_that("Dataset attributes", {
+        expect_identical(start_date(test.ds), NULL)
```

Comment:
But it isn't NULL. You need to `loadDataset` one of the other two datasets that have NULLs. I recommend using "ECON.sav" because it is not archived.